src/components/progress/test/basic/pages/page-one/page-one.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: 'page-one.html',
+  styles: [`
```

Comment:
Can you please instead move this style to a diffrent file, `page-one.scss`?
Also, note you need a space before `{` and after `:`